src/core/textrenderer/qgstextrenderer.h

```diff
@@ -144,6 +144,41 @@ class CORE_EXPORT QgsTextRenderer
                           QgsRenderContext &context, const QgsTextFormat &format,
                           bool drawAsOutlines = true );
 
+    /**
+     * Draws text along a line using the specified settings.
+     *
+     * \param line line to render text along
```

Comment:
The line points are defined in painter unit, correct? Should we specify in the doc or self-obvious? Same Q for the next function.

Reply:
I don't think it's harmful to be overly verbose even for "obvious" things! If you're unsure, then a mere mortal will definitely be unsure!